conan/tools/cmake/toolchain/blocks.py

```diff
@@ -518,12 +553,15 @@ def context(self):
             find_package_prefer_config = "OFF"
 
         is_apple_ = is_apple_os(self._conanfile)
+        is_win = self._conanfile.settings.get_safe("os") == "Windows"
```

Comment:
Don't be afraid of do the ``is_win`` inside the ``_get_host_runtime_dirs_mc(is_win, host_req)``, if no one else is needing the ``is_win`` variable in this ``context()`` method, then better make it internal to ``_get_host_runtime_dirs_mc()``.